manifests/integrations/kubernetes.pp

```diff
@@ -0,0 +1,45 @@
+# Class: datadog_agent::integrations::kubernetes
+#
+# This class will install the necessary configuration for the kubernetes integration
+#
+# Parameters:
+#   $url:
+#     The URL for kubernetes API
+#
+#   $tags:
+#     optional array of tags
+#
+#
+#
+# Sample Usage:
+#
+#   class { 'datadog_agent::integrations::kubernetes' :
+#     api_server_url  => 'https://kubernetes:443',
+#   }
+#
+class datadog_agent::integrations::kubernetes(
+  $api_server_url = 'Enter_Your_API_url',
+  $tags = []
+) inherits datadog_agent::params {
+  include datadog_agent
+
+  if $::datadog_agent::agent6_enable {
+    $dst = "${datadog_agent::conf6_dir}/kubernetes.yaml"
+  } else {
+    $dst = "${datadog_agent::conf_dir}/kubernetes.yaml"
```

Comment:
Dont forget kube_state_metrics as well.

Reply:
This is technically a "different" integration, would need it's own manifest.

Reply:
Ah I see. I'll add the manifest for it.